### ISSUE TITLE
Data for Aleutians survey 2024

### ISSUE THREAD

**Issue by lara-maleen** [afsc-gap-products/data-requests#107]
Hello, 

I collected sponges during the Groundfish survey in the Aleutian Islands (AI) this summer (Leg 1 and 2) on the R/V Ocean Explorer. I would like to request the associated survey data, such as date, coordinates/sites, depth, temperature at bottom, gear, substrate type (if possible) for each haul.  

You can also reach me via e-mail: 
lara.maleen.beckmann@gu.se

Thank you
Lara Beckmann

**Comment by MargaretSiple-NOAA** [comment]
Hi Lara-- You should be able to get all this data from FOSS: https://www.fisheries.noaa.gov/foss/f?p=215:28:14328612075185:::::
Leave another comment here if you cannot get what you need from there and we will assist you further! Thank you for your patience.

**Comment by lara-maleen** [comment]
Hi Margaret,
Thank you! I can see the haul data, but is it possible to download the data (e.g. as a csv file) from there? 

/Lara

**Comment by MargaretSiple-NOAA** [comment]
Yep-- once you run the report and can see the preview table of your data at the bottom, there will be a button at the top of that preview table that says "Actions." "Download" is one of them and once you click it you can choose what format you want the data in:
![Image](https://github.com/user-attachments/assets/2e3f1cb3-db7d-4be6-b945-46a68d01d352)

**Comment by MargaretSiple-NOAA** [comment]
@lara-maleen did this address your need?

**Comment by lara-maleen** [comment]
Thanks @MargaretSiple-NOAA ! I was able to download the data now (the download button didn't show in one of my browsers ;))

One question: Haul # 97 is missing from the data (Vessel Ocean Explorer, Leg 2). I made a note that the catch was discarded for the fish, but since the catch was mainly benthos, I collected several sponge specimens from that haul. Is it possible to receive data for this station?

**Comment by MargaretSiple-NOAA** [comment]
Oh yeah, that was a bad tow so wouldn't be included in the FOSS set. I exported the haul info [here](https://drive.google.com/drive/folders/1-WsLjbXcoxHgAOGv9HUFr-AFKq4ZdYmD). Let me know if that covers everything you need! No substrate type but I think that's normal for a haul with a negative performance code.

**Comment by lara-maleen** [comment]
Perfect! Now I have all the data I need. Thanks a lot!

**Comment by MargaretSiple-NOAA** [comment]
Awesome! Thanks for the quick response, and looking forward to seeing what you guys do with the data!
